net-news/liferea/liferea-1.12.5a.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit gnome2 python-single-r1
+
+DESCRIPTION="News Aggregator for RDF/RSS/CDF/Atom/Echo feeds"
+HOMEPAGE="https://lzone.de/liferea/"
+SRC_URI="https://github.com/lwindolf/${PN}/releases/download/v1.12.5/${P}.tar.bz2"
```

Comment:
I didn't used ```${PV//a}``` so with next bump simple renaming will fail for sure and I will use ```${PV}``` here.